src/controllers/authenticationController.js

```diff
@@ -91,6 +91,7 @@ export default class AuthenticationController {
         req.query = {};
         req.query.entityID = user.spid_idp;
 
+        // $FlowFixMe
```

Comment:
what's the problem here?

Reply:
Actually our spid-passport package isn't covered by a flow-typed definition, we will have to create a new one.